cypress/e2e/tests/migration/dynamic-report/issues/allIssues/sort.test.ts

```diff
@@ -23,8 +24,7 @@ describe(["@tier2"], "Issues sort validations", function () {
         cy.intercept("GET", "/hub/application*").as("getApplication");
     });
 
-    it("BUG MTA-2067 - Source Analysis on daytrader app and its issues sorting validation", function () {
-        // Create Analysis instance in before hook
+    it("Source Analysis on daytrader application", function () {
```

Comment:
Source analysis on day trader app is a TC covered in analysis.test.ts .. This is for sorting the issues . 

Let's rename this to " Creating data for sorting"

Reply:
Done